ucsmsdk/utils/converttopython.py

```diff
@@ -1237,9 +1008,6 @@ def _generate_cmdlets(xml_string):
 
     if category == "configConfMos" or category == "configConfMo":
         cmdlet = _generate_config_conf_cmdlets(xml_string)
-    elif category in ["configResolveDn", "configResolveDns",
```

Comment:
@ragupta-git so if this code goes away, how does convert_to decode configResolveDn/configResolveDns/configResolveClass/configResolveClasses ?